apps/dsb-message-broker/.env.example

```diff
@@ -3,5 +3,6 @@ PORT=3000
 WITH_SWAGGER=true
 JWT_SECRET=secret
 PRIVATE_KEY=
+MB_DID=
```

Comment:
When introducing new env variables, please also add documentation in README.md

Reply:
added to README.md